packages/builder/src/components/portal/licensing/licensingBanners.js

```diff
@@ -141,6 +145,35 @@ const buildPaymentFailedBanner = () => {
   }
 }
 
+const buildUsersAboveLimitBanner = EXPIRY_KEY => {
+  const userLicensing = get(licensing)
+  return {
+    key: EXPIRY_KEY,
+    type: BANNER_TYPES.WARNING,
+    criteria: () => {
+      return (
+        get(admin).cloud &&
+        userLicensing.license.plan.model === Constants.PlanModel.PER_USER &&
+        userLicensing.isFreePlan &&
+        userLicensing.quotaUsage.usageQuota.users > 5
+      )
+    },
+    message: `Free plan changes - Users will be limited to ${
+      userLicensing.license.quotas.usage.static.users.value
+    } users ${dayjs(
+      userLicensing.license.quotas.usage.static.users.value.startDate
+    ).fromNow()}`,
```

Comment:
What would be shown when we go past the start date?
Probably we should hide the banner at that point, as we'll start to deactivate then